### DISCUSSION TITLE
Supabase CLI on ARM

### DISCUSSION THREAD

**Discussion by JungeWerther** [supabase/supabase#28916] [Questions]
Hi,

I'm trying to install supabase cli on ARM. I know you can install the package directly but I'd rather use a package manager.

Linuxbrew is not supported on ARM, but I don't see any documentation or mention anywhere for using e.g. apt (which is the standard to my knowledge)

Could anyone recommend me the best course of action in this situation?

Hope it's not off-topic. Thank you :)

**Comment by silentworks** [comment]
You can use `npx` or `npm`.

npx for global install
```sh
npx supabase # this should prompt you to install the CLI
``` 
or inside of your project use

```sh
npm i -D supabase
````
and then to use it you use
```sh
npx supabase [command_you_want_to_call]

# example
npx supabase start
```
